redis_json/src/c_api.rs

```diff
@@ -491,21 +571,15 @@ macro_rules! redis_json_module_export_shared_api {
                 ));
                 ctx.export_shared_api(
                     (&JSONAPI_CURRENT as *const RedisJSONAPI_CURRENT).cast::<c_void>(),
-                    REDISJSON_GETAPI_V1.as_ptr().cast::<c_char>(),
-                );
-                ctx.log_notice("Exported RedisJSON_V1 API");
```

Comment:
Why removing v1 and v2?

Reply:
Will revert.
Should be in a separate PR if/when we bundle JSON and Search together and can drop support for various version combinations.

Reply:
When/if this will happened, we can also remove the unneeded ctx you mentioned here: https://github.com/RedisJSON/RedisJSON/pull/1047#discussion_r1260775975